engine/api/repositories_manager.go

```diff
@@ -321,14 +320,15 @@ func (api *API) getReposFromRepositoriesManagerHandler() Handler {
 
 		var repos []sdk.VCSRepo
 		if !api.Cache.Get(cacheKey, &repos) || len(repos) == 0 {
-			log.Debug("getReposFromRepositoriesManagerHandler> loading from Stash")
-			repos, err = client.Repos()
+			var errRepos error
```

Comment:
BTW i would prefer renaming the err in the parent scope rather than this limited scope

Reply:
done